packages/ember-routing/lib/system/router.js

```diff
@@ -303,9 +303,26 @@ var EmberRouter = EmberObject.extend(Evented, {
 
   _doURLTransition(routerJsMethod, url) {
     var transition = this.router[routerJsMethod](url || '/');
-    return didBeginTransition(transition, this);
+    didBeginTransition(transition, this);
+    return transition;
   },
 
+  /**
+    Transition the application into another route. The route may
+    be either a single route or route path:
+
+    See [Route.transitionTo](http://emberjs.com/api/classes/Ember.Route.html#method_transitionTo) for more info.
+
+    @method transitionTo
+    @param {String} name the name of the route or a URL
+    @param {...Object} models the model(s) or identifier(s) to be used while
+      transitioning to the route.
+    @param {Object} [options] optional hash with a queryParams property
+      containing a mapping of query parameters
+    @return {Transition} the transition object associated with this
+      attempted transition
+    @public
```

Comment:
Lets land this fix without changing public API (docs here are great, but change `@public` to `@private`), then submit a separate documentation only PR that updates?